### PR TITLE
Add Usage to README

### PR DESCRIPTION
**Pull request by tomholford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 ## Application Description
 
-TODO
+Generate multiple offline wallets for several popular cryptocurrencies.
```
Comment (ashelkovnykov) [inline README.md]: I think we should clarify the terminology around "wallet" and "address". The way that I've been thinking about it is that the "wallet" is the file that's generated with a specific mnemonic (and possibly passphrase). A wallet will contain one or more "addresses", which are specific to a cryptocurrency and BIP-44 path.

Unfortunately, I don't currently use the terms wallet and address that way in the code. I should refactor `Wallet` and the derived classes to do so.

This is all preamble to explain why I think that the text should instead say:
```
Generate offline wallets for several popular cryptocurrencies.
```

File: README.md [hunk 2]
```diff
@@ -27,7 +29,43 @@ TODO
...
 
-TODO
+```sh
+java -jar ./build/core/libs/core.jar
+```
+
+### Usage
+
+```
+-a, --account
```
Comment (ashelkovnykov) [inline README.md]: Should either indent all or none of the parameter descriptions

**Review by ashelkovnykov — CHANGES_REQUESTED** [review]
Thanks for the contribution. Just a note that I consider this placeholder info - I plan to do a thorough write-up for the README at a further date (no offense intended).

**Comment by ashelkovnykov** [comment]
Please squash the commits into one.

File: README.md (hunk 1 above)
Comment (tomholford) [reply README.md]: Thanks for the clarification

**Comment by tomholford** [comment]
> Please squash the commits into one.

Done, thanks for the feedback

File: README.md [hunk 3]
```diff
@@ -27,7 +29,68 @@ TODO
...
+#### Example
+Generate a cold wallet for dogecoin with 10 derived addresses.
+
+```sh
+java -jar ./build/core/libs/core.jar \
+-d ~/.wallets/cold/doge.wal \
+-c DOGE \
+-n 10 \
+-m "your mnemonic phrase here" \
+-p "yourpassphrase"
```
Comment (ashelkovnykov) [inline README.md]: While it's possible to enter the password (and mnemonic) on the command line, the preferred method is to just provide the `-p` parameter. JCommander will then request the password without echoing the input to screen, which is much more secure.

Comment (ashelkovnykov) [reply README.md]: During my personal use, I found it useful to do the same for the `-m` parameter. For the mnemonic, JCommander will echo the input to screen as you type. I found it easiest to copy/paste a particular mnemonic, and enter the password manually.

File: README.md [hunk 4]
```diff
@@ -27,7 +29,68 @@ TODO
...
+```
+
+### Output
+
+The tool generates a plain text file with the mnemonic and *n* generated addresses for each selected coin. Unless a `-d` flag is provided, output is written by default to:
```
Comment (ashelkovnykov) [inline README.md]: It generates the mnemonic, *n* addresses for either the selected coin or all coins if in hot-wallet mode, and the BIP 44/49/84 path for each address.

File: README.md [hunk 5]
```diff
@@ -27,7 +29,68 @@ TODO
...
+
+#### Linux
+```
+$HOME/.wallets/$DATE.wal
+```
+#### Mac
+```
+$HOME/Library/Wallets/$DATE.wal
+```
```
Comment (ashelkovnykov) [inline README.md:93]: May as well add Windows - the default write location is:
```
%APPDATA%\%DATE%.wal
```

**Review by ashelkovnykov — CHANGES_REQUESTED** [review]
Additional minor feedback. Ready to merge after this.

**Review by ashelkovnykov — APPROVED** [review]
LGTM

**Pull request merged**